src/components/Directory/List.js

```diff
@@ -32,6 +32,13 @@ tools
 
 
 export const list = [
+  {
+    name: "Towards Liberty",
+    url: "https://towardsliberty.com/",
+    type: "education",
```

Comment:
```suggestion
    type: "merchants",
    subType: "education",
```
The way the list is constructed, `education` is a subtype of `merchants`.

Reply:
oh, sorry for missing this, thanks for the suggestion!